.github/workflows/ci_test-full.yml

```
@@ -34,8 +34,8 @@ jobs:
 
     - name: Update pip
       run: |
-        # todo: unfreeze PIP after resolving minimal dependencies
-        pip install --quiet "pip==20.1" --upgrade --user  # needed for get pip cacher folder
+        # needed for `pip cache` command
+        pip install --quiet "pip>=20.1" --upgrade --user
```

```suggestion
        pip install --quiet pip --upgrade --user
```
I believe it works without specifying `>=20.1`, but not sure which is better.

---

I guess it works as the base image/agent already has it...